deploy/canary/Dockerfile

```diff
@@ -3,8 +3,7 @@ MAINTAINER vmarmol@google.com
 
 RUN apt-get install -y git dmsetup
```

Comment:
Can we just delete this file?  `make docker` seems sufficient for local development and uses `deploy/Dockerfile` instead.